packages/react-router/src/useNavigationItems.ts

```diff
@@ -0,0 +1,8 @@
+import { Runtime } from "./runtime.ts";
+import { useRuntime } from "@squide/core";
+
+export function useNavigationItems() {
+    const runtime = useRuntime() as Runtime;
```

Comment:
should you create a centralized hook useReactRouterRuntime instead of casting it everytime?

Reply:
I would prefer avoiding naming things with "ReactRouter" as it would be confusing. Some people might thing it's the actuall React Router Runtime not a Runtime from the shell. I don't expect `useRuntime` to be used often so it seems ok IMO to rely on a cast for this one.